### ISSUE TITLE
Perl Debugger hanged - Subroutine DB::postponed redefined

### ISSUE THREAD

**Issue by osintaj** [raix/vscode-perl-debug#142]
Hi,

I just installed visual studio code  and perl-debug under Linux Mint 19.1. Debugger started once once OK today but later it stopped with error:

ondrej@ondrej-VirtualBox:~/migrations/$ perl -d bin/map.pl -t PAYG_MAP 
Subroutine DB::postponed redefined at (eval 26)[/usr/share/perl/5.26/perl5db.pl:738] line 2.

And it is hanged there. Reboot of Linux machine didn't helped. I upgraded whole linux as well. I reinstalled this extension too. No help. Any hints?

Thank you,
Ondrej

**Comment by GiulioCesare** [comment]
I have exactly the same problem.
Hope you fix it soon.

Thanks, 
Giulio

**Comment by raix** [comment]
I suspect this to be an issue also when running the `perl -d` command?

**Comment by osintaj** [comment]
Right, I wanted to test debugging. Recently I tried again and it worked for some time again.
